dhis-2/dhis-services/dhis-service-core/src/main/java/org/hisp/dhis/scheduling/DefaultJobConfigurationService.java

```diff
@@ -94,16 +95,20 @@ public class DefaultJobConfigurationService implements JobConfigurationService {
   private final FileResourceService fileResourceService;
   private final SystemSettingManager systemSettings;
 
+  // TODO: MAS: investigate use of propagation req. new and failing tests
   @Override
-  @Transactional(propagation = Propagation.REQUIRES_NEW)
+  //    @Transactional(propagation = Propagation.REQUIRES_NEW, noRollbackFor =
+  //        Exception.class)
+  @Transactional
```

Comment:
i know this was brought up at the meeting this morning but i thought that we were leaving the requires new in the production code & trying to find a fix in the tests instead?

Reply:
This is updated now, class now checks for running profiles and changes propagation if it's a test running.